src/couch_views/src/couch_views_indexer.erl

```diff
@@ -46,45 +46,34 @@ init() ->
     Data = upgrade_data(Data0),
     #{
         <<"db_name">> := DbName,
+        <<"db_uuid">> := DbUUID,
         <<"ddoc_id">> := DDocId,
         <<"sig">> := JobSig,
         <<"retries">> := Retries
     } = Data,
-
```

Comment:
You changed whitespace in a code change PR.

Reply:
doh